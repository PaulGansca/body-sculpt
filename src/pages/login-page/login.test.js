import { shallow } from 'enzyme';
import React from 'react'
import LoginPage from './login-page';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<LoginPage />);
})

describe('<LoginPage />', () => {
  it("expect to render login page", () => {
    expect(wrapper).toMatchSnapshot();
  })
});