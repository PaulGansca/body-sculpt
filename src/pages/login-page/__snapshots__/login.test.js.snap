// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> expect to render login page 1`] = `ShallowWrapper {}`;
